metric/doc.go

```diff
@@ -68,6 +68,37 @@ asynchronous measurement, a Gauge ([Int64ObservableGauge],
 See the [OpenTelemetry documentation] for more information about instruments
 and their intended use.
 
+# Measurements
+
+Measurements are made by recording values and information about the values with
+an instrument. How these measurements are recorded depends on the instrument.
+
+Measurements for synchronous instruments ([Int64Counter], [Int64UpDownCounter],
+[Int64Histogram], [Float64Counter], [Float64UpDownCounter], [Float64Histogram])
+are made using the instrument methods directly. All counter instruments have an
+`Add` method that is used to measure an increment value, and all histogram
+instruments have a `Record` method to measure a data point.
+
+Asynchronous instruments ([Int64ObservableCounter],
+[Int64ObservableUpDownCounter], [Int64ObservableGauge],
+[Float64ObservableCounter], [Float64ObservableUpDownCounter],
+[Float64ObservableGauge]) make measurements within a callback function. The
+callback is registered with the Meter which ensures the callback is called once
+per collection cycle. A callback can be registered two ways: during its
+creation using an option; the RegisterCallback method of the Meter that created
+the instrument.
```

Comment:
Is it possible—even if inadvisable—to use both of these techniques together, both registering a callback at instrument creation time _and_ registering one or more callbacks on the `Meter`?
```suggestion
per collection cycle. A callback can be registered in either of two ways: during the instrument's creation using an option, or later with the `RegisterCallback` method of the Meter that created the instrument.
```

Reply:
Sure. If you want to measure different distinct measurements for the instrument where one measurement meets the criteria and the other doesn't (i.e. multi-instrument callback) there is reason to do so.

Reply:
Updated, PTAL.